packages/material-ui-lab/src/TreeItem/TreeItem.js

```diff
@@ -6,41 +6,53 @@ import Typography from '@material-ui/core/Typography';
 import Collapse from '@material-ui/core/Collapse';
 import { withStyles } from '@material-ui/core/styles';
 import { useForkRef } from '@material-ui/core/utils';
+import { useIsFocusVisible } from '../../../material-ui/src/utils/focusVisible';
 import TreeViewContext from '../TreeView/TreeViewContext';
 
-export const styles = {
-  /* Styles applied to the root element. */
-  root: {
-    listStyle: 'none',
-    margin: 0,
-    padding: 0,
-    outline: 0,
-    '&:focus > $content': {
-      outline: 'auto 1px',
+export const styles = function styles(theme) {
+  return {
+    /* Styles applied to the root element. */
+    root: {
+      listStyle: 'none',
+      margin: 0,
+      padding: 0,
+      outline: 0,
+      cursor: 'pointer',
```

Comment:
+1 for the cursor style, I was "confused" when interacting the component without. I have added it too in https://github.com/mui-org/material-ui/pull/16785/files#diff-8bf52f8ffafb5d687f06556db6747f41R30.

However, I think that it should be applied to the content element, where we apply the click listener.

Reply:
+1 for adding it to the content element